.claude/skills/github/scripts/pr/Add-PRReviewThreadReply.ps1

```diff
@@ -0,0 +1,163 @@
+<#
+.SYNOPSIS
+    Adds a reply to a GitHub PR review thread using GraphQL.
+
+.DESCRIPTION
+    Posts a reply to a review thread using the thread ID (PRRT_...) rather than
+    comment ID. This is required for proper thread management when using
+    branch protection rules.
+
+    Optionally resolves the thread after posting the reply.
+
+.PARAMETER ThreadId
+    The GraphQL thread ID (e.g., PRRT_kwDOQoWRls5m3L76).
+
+.PARAMETER Body
+    Reply text (inline). Mutually exclusive with BodyFile.
+
+.PARAMETER BodyFile
+    Path to file containing reply. Mutually exclusive with Body.
+
+.PARAMETER Resolve
+    If specified, resolves the thread after posting the reply.
+
+.EXAMPLE
+    ./Add-PRReviewThreadReply.ps1 -ThreadId "PRRT_kwDOQoWRls5m3L76" -Body "Fixed in abc1234."
+
+.EXAMPLE
+    ./Add-PRReviewThreadReply.ps1 -ThreadId "PRRT_kwDOQoWRls5m3L76" -Body "Fixed." -Resolve
+
+.EXAMPLE
+    ./Add-PRReviewThreadReply.ps1 -ThreadId "PRRT_kwDOQoWRls5m3L76" -BodyFile ./response.md
+
+.NOTES
+    Exit Codes: 0=Success, 1=Invalid params, 2=File not found, 3=API error, 4=Not authenticated

```

Comment:
@rjmurillo-bot the exit codes for all the scripts need to be 1) well documented (outside of the documentation of the script, like in an ADR), and 2) consistent.